docs/dev_guide.rst

```diff
@@ -0,0 +1,125 @@
+.. _dev_guide:
+
+*****************
+Developer's Guide
+*****************
+
```

Comment:
There should be a reference or link to the installing the development repository. 

Also a note that they should fork the repo on github if they want to contribute.

Reply:
Can you check if I did it right?

Reply:
Did the links render correctly, if so, yes, if not, no.

Reply:
Yes they work.